app/controllers/sections/info/LocalReferenceNumberController.scala

```diff
@@ -88,6 +89,22 @@ class LocalReferenceNumberController @Inject()(
         Future.successful(Redirect(controllers.sections.info.routes.DeferredMovementController.onPageLoad(request.ern)))
     }
 
+  private def withDispatchPlaceAnswer(f: DispatchPlace => Future[Result])(implicit request: UserRequest[_]): Future[Result] =
```

Comment:
I don't like this. As far as I know, the Dispatch Place question should only be asked to XIWK people. It is intentional that no other trader types are asked this question.